runtime/lua/vim/treesitter/query.lua

```diff
@@ -242,7 +242,14 @@ local directive_handlers = {
     if range[1] < range[3] or (range[1] == range[3] and range[2] <= range[4]) then
       metadata[pred[2]][key] = range
     end
-  end
+  end,
+
+  ["downcase!"] = function(match, _, bufnr, pred, metadata)
```

Comment:
Could this be `lowercase` instead? I know it is minor but some people may find that weird.

Reply:
No, that's the standard lisp name. Lisp are just a bit weird.

Reply:
This is a sexpr DSL within lua and and not common lisp tho.